docs/categories/index.html

```diff
@@ -1244,6 +1263,22 @@ <h1>
             
         
                     
+                        
+            
+            
+                
+                    
+                
+                
+
+                    
+                    
+                    
+
+                    
+            
+        
+                    
```

Comment:
That's the new version of Hugo? (I thought only the old one makes those whitespaces.)

Reply:
Yes, it's `hugo-0.69.0` :-(